src/lib/containers.ts

```diff
@@ -0,0 +1,154 @@
+import * as Logging from "../logging"
+const logger = new Logging.Logger("excmd")
+
+/** Creates a container from the specified parameters. Unlike the official Mozilla addon, does not
+ *  allow multiple  containers with the same (name, color, icon) constraint.
+ *  @param containerName  The container name.
+ *  @param containerColor  The container color, limited to browser.contextualIdentities.IdentityColor.
+ *  @param containerIcon  The container icon, limited to browser.contextualIdentities.IdentityIcon.
```

Comment:
The help should list the valid colours / icons, or at least link to the MDN page.